Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json

```diff
@@ -47,7 +47,7 @@
           "ManualInstructions": [],
           "args": {
             "referenceType": "package",
-            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.84-preview",
+            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.112-preview",
```

Comment:
112 or 122?

Reply:
fixed that, thanks!!

Reply:
still shows as 112 for input bindings here. is that intentional?